hw1/ex4.py

```diff
@@ -0,0 +1,4 @@
+print ("Введи кол-во журавликов")
+s = int(input())
+print ("Каждый сделает по:")
+print(int(s / 6), int(s * 2 / 3), int(s / 6))
```

Comment:
Функцию `int` для получения только целой части числа использовать можно, но зачем, если есть уже соответствующий оператор?
```suggestion
print(s // 6, s * 2 // 3, s // 6)
```

Reply:
А еще вы дважды выполняете одну и ту же операцию. Чтобы такого не происходило используют переменные.